lib/project_types/theme/commands/package.rb

```diff
@@ -53,6 +56,18 @@ def theme_name(path)
 
         [theme_name, theme_info["theme_version"]].compact.join("-")
       end
+
+      def command
+        @command ||= if @ctx.which(ZIP) == true
+          ZIP
+        elsif @ctx.which(SEVEN_ZIP) == true
+          SEVEN_ZIP
+        end
```

Comment:
soooo originally i had this:
```
@command ||= (if @ctx.which(ZIP) == true
  ZIP
elsif @ctx.which(SEVEN_ZIP) == true
  SEVEN_ZIP
end)
```

but this was not rubocop friendly. without the parentheses, the `||=` statement doesn't work as it should and it evaluates the if statement over and over again. im not sure why?